bitnami/fluentd/values.yaml

```diff
@@ -39,6 +39,15 @@ clusterDomain: cluster.local
 
 forwarder:
   enabled: true
+
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 0
+    fsGroup: 0
+
```

Comment:
On the `README` you are saying that the default values are 1001. I assume the `README` is wrong.